shell/lib/github.sh

```diff
@@ -1,9 +1,13 @@
 #!/usr/bin/env bash
 # Contains various helper functions for interacting with Github.
-#
-# Requires:
-# - asdf.sh
-# - logging.sh
+
+# LIB_DIR is the directory that shell script libraries live in.
```

Comment:
Note the `LIB_DIR` convention to further try to reduce issues with the `DIR` env var that is used by other scripts.

We really need to move to Go.